docs/modules/operation/pages/flows/setup.adoc

```diff
@@ -16,7 +16,7 @@ From a Karaf shell on your {page-component-title} instance, start by configuring
 $ ssh -p 8101 admin@localhost
 ...
 admin@opennms()> config:edit org.opennms.features.flows.persistence.elastic
-admin@opennms()> config:property-set elasticUrl http://elastic:9200
+admin@opennms()> config:property-set elasticUrl http://${YOUR_ELASTICSEARCH_URL}:9200
```

Comment:
I'd use `YOUR_ELASTICSEARCH_HOSTNAME_IP_OR_FQDN`, as a URL is what the whole `elasticUrl` expects to have (not what the placeholder appeals to be). The problem with using `${}` is, that might be misleading. Or, explain that `elastic` is an example of the hostname, IP Address, or FQDN which is what you could use there. In fact, it can be a list of URLs.

Reply:
Thanks @agalue for your suggestion. I've updated it accordingly.